### PR TITLE
INOUE - update property and enum terms yaml files

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: src/gdcdictionary/schemas/_terms.yaml [hunk 1]
```diff
@@ -7362,10 +7365,137 @@ country_of_birth:
...
+      cde_id:
+      cde_version:
+      term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C198198"
+      term_id: C198198
+      term_version: 23.08d
+
+oxygen_use_type:
+  common:
+    description: >-
+      Text term to decribe the type of oxygen the patient uses at home.
```
Comment (clsackett) [inline src/gdcdictionary/schemas/_terms.yaml:7469]: typo: change "decribe" to "describe"

**Review by clsackett — COMMENTED** [review]
One small typo otherwise approved!

File: src/gdcdictionary/schemas/_terms.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml]: updated 👍

**Review by wwysoc2 — APPROVED** [review]
lgtm

**Comment by wwysoc2** [comment]
https://gdc-ctds.atlassian.net/browse/DICT-351

**Review by wrkhenddher — APPROVED** [review]
All build checks pass so it's hopefully all good :)

**Pull request merged**